charts/externalcreds/templates/podDisruptionBudget.yaml

```diff
@@ -0,0 +1,8 @@
+{{- include "libchart.pdb" (list . "externalcreds.pdb" ) }}
+
+{{- define "externalcreds.pdb" -}}
+spec:
+  selector:
+    matchLabels:
+      deployment: {{ $.Values.name }}-{{ .name }}-deployment
```

Comment:
I'm not sure this line will render correctly with the way the chart is currently set up (two deployments)

Reply:
a747ad95